### ISSUE TITLE
What's the shape of input?

### ISSUE THREAD

**Issue by mikumeow** [automan000/Convolution_LSTM_PyTorch#12]
This is a question rather than an issue, sorry for bothering. > +
I have questions about input/output shape, as well as the meaning of "input_channel".
According to #gradient check part of the code, the input is in the shape of (1, 512, 64, 32), while the output shrinks to (1, 32, 64, 32).
I assume for the input, 1 is batch size,  512 is input_channel, and the image is of size 64*32.

The questions are: What does these channels mean? Are they filters as in Keras convLSTM library(https://keras.io/layers/recurrent/#convlstm2dcell)? How do we input a sequence of 5 images? And why is output channel smaller than the input?

**Comment by mikumeow** [comment]
Have another issue after further understanding and after author updated the code.
